projects/common/src/utilities/formatters/string/string-formatter.ts

```diff
@@ -25,11 +23,11 @@ export const displayString = (provided?: unknown): string => {
         ? 'Unknown'
```

Comment:
a stray unknown (didn't show up in tests because this is unreachable - nullness is checked on line 14 so we can remove this nested ternary.

Reply:
done